app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,10 @@
+class AdminController < ApplicationController
+
+  def index
+    @users = User.where.not(username: current_user.username).order(created_at: :asc).page(params[:page])
```

Comment:
it's better to exclude by id

Reply:
Yes it is indeed! But since the logic implemented for soft deleting users. There are empty id's. So since the username is unique I pick it 😊

Reply:
I think you need to add (deactivated: false) to @users to solve the pagination problem. I'm not sure if @users goal is to get active_users only